plugins/graphql-kotlin-gradle-plugin/src/main/kotlin/com/expediagroup/graphql/plugin/gradle/config/GraphQLParserOptions.kt

```diff
@@ -0,0 +1,20 @@
+package com.expediagroup.graphql.plugin.gradle.config
+
+import java.io.Serializable
+
+/**
+ * Configure options for parsing GraphQL queries and schema definition language documents. Settings
+ * here override the defaults set by GraphQL Java.
+ */
+data class GraphQLParserOptions(
+    /** Modify the maximum number of tokens read to prevent processing extremely large queries */
+    var maxTokens: Int? = 15000,
```

Comment:
if those have default values set -> does it make sense to keep those fields as nullable?

Reply:
If you wish to go back to the default value (whatever it is) then you can explicitly set it as null.

Reply:
seems odd experience for the end user - if the intent is to keep null as not-set and use defaults from `graphql-java` lets revert back to keep those values null and document the behavior in the docs